http3/server_test.go

```diff
@@ -201,6 +201,36 @@ var _ = Describe("Server", func() {
 			Expect(hfs).To(HaveKeyWithValue(":status", []string{"200"}))
 		})
 
+		It("handles aborts", func() {
```

Comment:
This was just a simple test that I'm not 100% sure is testing what is needed to be tested. But it does test part IMHO.

Reply:
The test below this should be sufficient.

Reply:
Cool, thanks!

Reply:
Can you remove this one?

Reply:
Done.